### DISCUSSION TITLE
Using Egui with Smithay Client Toolkit

### DISCUSSION THREAD

**Discussion by Arthex1** [emilk/egui#3142] [Q&A]
Hello, I am new to using egui and trying to use it with sctk. As far as I can understand, egui needs input(keyboard and mouse) and then outputs some data, which can be rendered to the screen. But how is it done? are there any examples i could refer to?

that's all, thanks!

**Comment by emilk** [comment]
You can take a look at https://github.com/not-fl3/egui-miniquad or one of the many other integrations: https://github.com/emilk/egui#integrations
